leap/README.md

```diff
@@ -0,0 +1,40 @@
+## leap: meta-gradient path learner in MXNet
+
+--------------------------------------------------------------------------------
+
+MXNet implementation of  "leap", the meta-gradient path learner published in ICLR 2019: [(link)](https://arxiv.org/abs/1812.01054) by S. Flennerhag, P. G. Moreno, N. Lawrence, A. Damianou.
+
+
+## Getting Started
+Check the `demos` and `test` folders. 
```

Comment:
Consider adding link to installing jupyter/jupyter lab.

The mxnet_test seems more like a demo than a test since no checks are done and it cannot be run with any normal test runner (e.g. pytest, unittest). Maybe it can move to the demo folder?

To run the demo notebook, the user currently needs to first install leap but you can get around it with either changing the directory or adding the leap directory to the pythonpath. There doesn't really seem to be a general convention but xfer-ml requires the the package to be installed which I think is fine to continue as a convention for the repo.